kernel/src/path.rs

```diff
@@ -234,8 +234,25 @@ mod tests {
 
         // ignored - no extension
         let log_path = table_log_dir.join("00000000000000000010").unwrap();
-        let log_path = ParsedLogPath::try_from(log_path).unwrap();
-        assert!(log_path.is_none());
+        let result = ParsedLogPath::try_from(log_path);
+        assert!(
+            matches!(result, Ok(None)),
+            "Expected Ok(None) for missing file extension"
+        );
+
+        // empty extension - should be treated as unknown file type
+        let log_path = table_log_dir.join("00000000000000000011.").unwrap();
+        let result = ParsedLogPath::try_from(log_path);
+        assert!(
+            matches!(result, Ok(Some(_))),
+            "Expected Ok(Some) for empty extension"
+        );
+        if let Ok(Some(parsed)) = result {
+            assert!(
+                parsed.is_unknown(),
+                "Expected Unknown file type for empty extension"
+            );
+        }
```

Comment:
Any reason not to just `.unwrap().unwrap()` since this is a unit test that should hopefully never fail? Alternatively, we could do something like:
```rust
let result = ParsedLogPath::try_from(log_path);
let Ok(Some(ParsedLogPath { file_type: Unknown, .. })) = result else {
    panic!("Expected Unknown file type, got {:?}", result);
};
```
or just 
```rust
assert!(
    matches!(result, Ok(Some(ParsedLogPath { file_type: Unknown, .. })),
    "Expected Unknown file type, got {result:?}"
);
```